parser_library/test/workspace/extension_handling_test.cpp

```diff
@@ -62,7 +62,7 @@ TEST(extension_handling_test, extension_removal)
 
     // test no extensions
     library_local lib5(file_mngr, lib2_loc, { {} }, empty_loc);
-    EXPECT_FALSE(lib5.has_file("MAC"));
+    EXPECT_TRUE(lib5.has_file("MAC"));
```

Comment:
I might be missing something here but I would have thought that "MAC" would not be found at all for `lib2_loc`?

I don't know what was the original idea behind this but I think `list_directory_files()` in `file_manager_extension_mock` should return empty result for any other request than `lib_loc` now.

The same goes for the rest of the lines where `lib2_loc` is used.

Reply:
I will remove the uses of lib2_loc because they are not really used correctly in the test and matching the library prefix is not the point of the test anyway.